docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,281 @@
+---
+jupytext:
+ text_representation:
+   extension: .md
+   format_name: myst
+   format_version: 0.13
+   jupytext_version: 1.11.1
+kernelspec:
+ display_name: Python 3
+ language: python
+ name: python3
+---
+
+
+# What additional options are available when using LRE?
+
+
+In [](lre-1-intro.md), {func}`.execute_with_lre` was used to calculated the error mitigated expectation values
+of an input circuit with the default parameters. In this section we will detail the optional arguments this technique can be used, and tuned with.
+
+
+
+
+```
+from mitiq.lre import execute_with_lre
+
+
+lre_value = execute_with_lre(
+   circuit,
+   executor,
+   degree,
+   fold_multiplier,
+   folding_method = <"noise scaling method imported from zne.scaling.folding">,
+   num_chunks = <"group a large circuit into a circuit with fewer layers">
+)
+```
+
+
+
+
+The options that can be used to control the hyperparameters are:
+
+
+- `degree` of the extrapolating polynomial
+- `fold_multiplier` to control how a circuit is scaled
+- `folding_method` to choose the unitary folding method
+- `num_chunks` to alter the sampling cost
+
+
+## Controlling how the noise-scaled circuits are created
+
+
+### Extrapolating polynomial
+
+
+Depending on the choice of the degree of the polynomial, how each noise-scaled circuit is scaled varies. This
+in turn influences how the sample matrix for multivariate Richardson extrapolation is created.
+
+
+Suppose we have an input circuit of 4 layers.
+
+
+```{code-cell} ipython3
+from cirq import LineQubit, Circuit, CZ, CNOT, H
+
+
+q0, q1, q2, q3  = LineQubit.range(4)
+circuit = Circuit(
+   H(q0),
+   CNOT.on(q0, q1),
+   CZ.on(q1, q2),
+   CNOT.on(q2, q3),
+)
+
+
+print(circuit)
+```
+
+
+
+
+How the circuits are scaled is controlled by `degree`, `fold_multiplier` and number of layers in the input circuit. We choose
+`degree = 2` and `fold_multiplier=2` for demonstration purposes.
+
+
+```{code-cell} ipython3
+degree = 2
```

Comment:
Seems odd to define them here if we just hardcode these values directly as arguments to the function later.

Reply:
yes, it's a leftover. I was planning to define variables for the values I used, scrapped that idea.